service/src/main/java/uk/gov/hmcts/reform/fpl/model/ApplicantParty.java

```diff
@@ -8,15 +8,24 @@
 import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
 import uk.gov.hmcts.reform.fpl.model.common.Party;
 import uk.gov.hmcts.reform.fpl.model.common.Telephone;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantContactName;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantTelephone;
 
 import java.util.Date;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
+@HasApplicantTelephone
+@HasApplicantContactName
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class ApplicantParty extends Party {
+    @NotBlank(message = "Enter the applicant's full name")
```

Comment:
Probably more a user question but if this is always an organisation name should it read enter the applicants full name or possibly something like enter the applicant/organisation name :) probably not but just a question.

Reply:
The label on the UI reads "Name of the applicant" organisation is not mentioned.